cypress-tests/cypress/support/redirectionHandler.js

```diff
@@ -295,7 +295,7 @@ function threeDsRedirection(redirection_url, expected_url, connectorId) {
         cy.get('input[type="password"]').type("password");
         cy.get("#buttonSubmit").click();
       });
-  } else if (connectorId === "bankofamerica" || connectorId === "cybersource") {
+  } else if (connectorId === "bankofamerica" || connectorId === "cybersource" || connectorId === "wellsfargo"){
```

Comment:
Please add a screenshot of tests running successfully.

Reply:
Keeping this `{|| connectorId === "wellsfargo")}` for now, Three_DS flow is passing through non_three_ds flow